### ISSUE TITLE
Don't limit duplicates by default

### ISSUE THREAD

**Issue by wrmacrae** [Senryoku/MTGADraft#98]
In Survey #1, 78.6% of respondents prefer that the default does not limit duplicates in a pod. I think the question was a bit flawed, but regardless we should change the default over.

**Comment by tooomm** [comment]
The limit cards per rarity setting looks highly advanced and not something users want to adjust frequently.
I can hardly believe users prefer an all-random card pool over a more realistic experience which limiting provides due to the underlying print runs wizards uses in real boosters?!

Maybe implement a reasonable exponential decreasing chance of duplicates and remove the option completely? So that more than 2 rares or 1 mythic of a type are not impossible but normally don't happen. (But have a tick box with "100% randomized cards within each slot for each booster"?)

**Comment by wrmacrae** [comment]
> The limit cards per rarity setting looks highly advanced and not something users want to adjust frequently.
> I can hardly believe users prefer an all-random card pool over a more realistic experience which limiting provides due to the underlying print runs wizards uses in real boosters?!
> 
> Maybe implement a reasonable exponential decreasing chance of duplicates and remove the option completely? So that more than 2 rares or 1 mythic of a type are not impossible but normally don't happen. (But have a tick box with "100% randomized cards within each slot for each booster"?)

I agree that this is a fairly advanced thing that people won't want to fiddle with much. It's especially hard if we change the defaults to "no duplicate limitation" by changing the default numbers to 99 for each, because then people won't have a quick sense of what numbers to punch in to provide some limitation, and so for people that think this is an important feature, maybe the right thing is to wrap up the current values in a single checkbox that just says "Limit duplicates in pod" and explains the details on hover. This takes away some granularity of control, but speeds things up, provides a suggestion, and save a bunch of UI space.

As for user preference, here's the preferences from the survey we ran:
<img width="920" alt="Screen Shot 2020-09-07 at 2 28 24 PM" src="https://user-images.githubusercontent.com/2793499/92417321-6ab45880-f116-11ea-99b9-19e4e6936e5c.png">
8 out of 43 respondents (18.6%) wanted to limit duplicates by default, whereas the other 81.4% indicated that they want the default changed, they don't want a duplicate limit option at all, or they don't think this is representative of real drafting. Without having laid out the questions perfectly (I should have explained the current performance, and made the options a more balanced set of "A only, A by default with an option for B, option between A or B and default doesn't matter to me, B by default with an option for A, B only, no opinion"), I'm willing to accept this was skewed a little bit. But still, the overwhelming proportion of users _do_ prefer an all-random card pool (that is, individual booster won't have duplication within one pack, but pods may).

As for what really happens: cognitive biases around randomized results can be very strong, especially when individuals have emotional stakes in the situation, so I suspect most people have stronger opinions and convictions on this topic than they should, and that without careful analysis most people overestimate how knowledgable they are about the reality. My own approach to evaluation how Arena probably solves this problem is to imagine how I would have implemented it, and I think that makes it likely to be either A) completely random because that's the easiest thing to execute, or B) the same as paper, because there's value to resembling the paper game, but I lean toward A because it would be way less effort and developers tend to be short on time, because they might not have access to the information on the print colation early enough in the development cycle, and because Occam's Razor basically says we should lean toward it being the simplest process vs attributing complexity wee don't have evidence for. I think it's most important that we imitate what would happen in Arena versus what happens in paper, but paper is worth considering because there's a chance Arena is modeled after that.

For paper, I know that collation/print rules historically meant boxes were not purely random, to the point that with older sets you could predict what would be in some packs based on others. But I also have the impression that over time the boxes have approached random (hence why we don't see that box mapping as a big problem in retail today), and we've heard anecdotes of 3 of the same rare in one box. Also, if you draft with packs from different boxes, then there's certainly no duplicate prevention.

In summary, I could be persuaded to mimic (and document) Arena's performance in the app if someone determined what that performance is with data we can see so that we can challenge the analysis. Without that, I think we should defer to user preference, which seems to be allowing duplicates to occur. As for exponentially decreasing chances, there's an emergent exponential decrease in the way the independent events and their probabilities work out, which I think is why players are surprised when they see duplicates, but I don't think we should outright prevent them or further lower their likeliness.

**Comment by Senryoku** [comment]
> For paper, I know that collation/print rules historically meant boxes were not purely random, to the point that with older sets you could predict what would be in some packs based on others. But I also have the impression that over time the boxes have approached random (hence why we don't see that box mapping as a big problem in retail today), and we've heard anecdotes of 3 of the same rare in one box. Also, if you draft with packs from different boxes, then there's certainly no duplicate prevention.

This is where I'm at right now. The limits were added to address concerns of players seeing 2 or 3 duplicate rares regularly. I knew it wasn't weird statistically speaking, but thought that maybe - having heard of box mapping and the like - that feeling was due to real life experience with retail boxes. Now I kinda doubt it.

Honestly I'm inclined to just double the default values. It will still convey enough meaning to be understood, leave the setting accessible for those who want it and won't affect regular drafting meaningfully.

**Comment by wrmacrae** [comment]
Doubling them is a pretty solid way to keep the options around and indicate rough proportions without really imposing much of any real limitation in practice. I think the biggest drawback would be when some users don't have an idea of how irrelevant those defaults are likely to be (and for other values they set, how restrictive or irrelevant those may be). A secondary drawback is that it's a lot of options and screen real estate to support this. So, if we do that, I'd suggest circling back on usage data after a few months, with the idea of removing the option at that time if it's under-utilized. Alternatively, I think just removing the options or wrapping them up into a single checkbox, defaulted off, that sets the current values ("limit duplicates") would also work well. But the change you mentioned or any of these two would be an improvement over current performance, so I'm relatively happy with any of them.

**Comment by Senryoku** [comment]
Doubled limits by default e67afea34da512fb0a391d082f2e88819b01c6d6

**Comment by wrmacrae** [comment]
Will call this done because I think the doubled limits effectively accomplish this
